toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/WorldTrackingCameraController.kt

```diff
@@ -114,31 +111,13 @@ internal class WorldTrackingCameraController(private val onLocationDataSourceFai
             Camera(
                 location.position.y,
                 location.position.x,
-                if (location.position.hasZ) location.position.z ?: totalElevationOffset else totalElevationOffset,
-                totalHeadingOffset,
+                if (location.position.hasZ) location.position.z ?: calibrationState.totalElevationOffset else calibrationState.totalElevationOffset,
+                calibrationState.totalElevationOffset,
```

Comment:
Shouldn't this apply the heading instead of elevation?
```suggestion
                calibrationState.totalHeadingOffset,
```